.github/actions/bionic/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:bionic
+
+LABEL maintainer "Hiroki Kumazaki <kumagi@google.com>"
+
+RUN grep -q "# deb-src" /etc/apt/sources.list && sed -Ei 's/^# deb-src /deb-src /' /etc/apt/sources.list || \
+    cat /etc/apt/sources.list | grep "^deb " | sed -e "s/^deb /deb-src /" | tee -a /etc/apt/sources.list && \
```

Comment:
I'm not sure it is safe to read and append to a file simultaneously.  Consider a temporary file or use `sponge`.  Also you don't need `cat` or `grep`.

Reply:
Done.